.github/workflows/chronograf-cypress-tests.yaml

```diff
@@ -0,0 +1,84 @@
+name: "chronograf/cypress"
+
+on:
+  pull_request:
+  schedule:
+    - cron: '30 5 * * 1-5'
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.4'
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.6.3
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        with:
+          cluster_name: "chronograf-testing"
+          config: .github/workflows/resources/kind-config.yaml
+
+      - name: Create Enterprise Test Resources
+        run: |
+          helm repo add jetstack https://charts.jetstack.io
+          helm repo add influxdata https://helm.influxdata.com/
+          helm repo update
+          helm upgrade --wait --install \
+              cert-manager jetstack/cert-manager \
+              --namespace cert-manager \
+              --create-namespace \
+              --version v1.5.4 \
+              --set prometheus.enabled=false \
+              --set webhook.timeoutSeconds=30 \
+              --set installCRDs=true
+          kubectl apply -f .github/workflows/resources/test-reources.yaml
+          kubectl create secret generic influxdb-license --from-literal=INFLUXDB_ENTERPRISE_LICENSE_KEY=${INFLUXDB_ENTERPRISE_LICENSE_KEY}
+          helm upgrade --install influxdb influxdata/influxdb-enterprise --namespace default --set-string envFromSecret=influxdb-license --set-string data.service.type=NodePort
+          kubectl patch svc influxdb-influxdb-enterprise-data --type=json -p '[{"op":"replace","path":"/spec/ports/0/nodePort","value":30086}]'
+        env:
+          INFLUXDB_ENTERPRISE_LICENSE_KEY: "${{ secrets.INFLUXDB_ENTERPRISE_LICENSE_KEY }}"
+
+      - name: Chronograf
+        run: |
+          npm install --global yarn
+          make
+          RUNNER_TRACKING_ID="" && (nohup ./chronograf > out.log 2>&1 &)
+          sleep 10
+
+      - name: Chronograf test url
+        run: |
+          echo "InfluxDB data node status: $(curl -Isk "https://localhost:8086/ping" | head -n 1)"
+          echo "Chronograf status: $(curl -Isk "http://localhost:8888" | head -n 1)"
+          cat out.log || true
+
+      - name: Cypress
+        uses: addnab/docker-run-action@v3
+        with:
+          image: cypress/included:9.5.2
+          options: -i -v ${{ github.workspace }}/ui:/ui -w /ui --add-host=host.docker.internal:host-gateway
+          shell: sh
+          run: |
+            yarn install
+            npx browserslist@latest --update-db
+            cypress run --browser chrome --config baseUrl=http://host.docker.internal:8888
```

Comment:
We should better running the same cypress that is in installed via yarn install (defined in `package.json`), which is `8.4.1`. Couldn't we do so rather than relying upon a custom image?

Reply:
It runs cypress in docker image "cypress/included" (version will be fixed - good point) because it needs browsers and dependencies already installed to run cypress in that browsers, so those requirements are satisfied by using ready-to-use image from cypress.